packages/site/src/main.css

```diff
@@ -1,5 +1,11 @@
 :root {
-  --sideBarWidth: 250px;
+  --sideBarWidth: 200px;
```

Comment:
We do end up with some wrapping here but I think it's a worthwhile tradeoff for optimizing doc content space